src/main/java/org/mskcc/oncokb/curation/OncokbCurationApp.java

```diff
@@ -173,20 +162,15 @@ private static void initSentry(Environment env) {
                 // Completed disable the traces sampling
                 options.setTracesSampler(null);
                 options.setTracesSampleRate(null);
-
                 // Ignore the following exceptions
-                options.addIgnoredExceptionForType(BadCredentialsException.class);
-                options.addIgnoredExceptionForType(IOException.class);
-                options.addIgnoredExceptionForType(HttpClientErrorException.class);
-                options.addIgnoredExceptionForType(HttpServerErrorException.class);
-                options.addIgnoredExceptionForType(ResponseStatusException.class);
-                options.addIgnoredExceptionForType(EmailAlreadyUsedException.class);
-                options.addIgnoredExceptionForType(InvalidPasswordException.class);
-                options.addIgnoredExceptionForType(UsernameAlreadyUsedException.class);
-                options.addIgnoredExceptionForType(HttpRequestMethodNotSupportedException.class);
-                options.addIgnoredExceptionForType(InternalAuthenticationServiceException.class);
-                options.addIgnoredExceptionForType(BadRequestAlertException.class);
-                options.addIgnoredExceptionForType(ResourceAccessException.class);
+                //                options.addIgnoredExceptionForType(IOException.class);
+                //                options.addIgnoredExceptionForType(HttpClientErrorException.class);
+                //                options.addIgnoredExceptionForType(HttpServerErrorException.class);
+                //                options.addIgnoredExceptionForType(ResponseStatusException.class);
+                //                options.addIgnoredExceptionForType(HttpRequestMethodNotSupportedException.class);
+                //                options.addIgnoredExceptionForType(InternalAuthenticationServiceException.class);
+                //                options.addIgnoredExceptionForType(BadRequestAlertException.class);
+                //                options.addIgnoredExceptionForType(ResourceAccessException.class);
```

Comment:
Let's keep these here for now. I will put them back based on the exceptions we receive in production.